bin/internal/shared.sh

```diff
@@ -143,8 +153,13 @@ function upgrade_flutter () (
 
     retry_upgrade
 
-    "$DART" --disable-dart-dev $FLUTTER_TOOL_ARGS --snapshot="$SNAPSHOT_PATH" --packages="$FLUTTER_TOOLS_DIR/.packages" --no-enable-mirrors "$SCRIPT_PATH"
-    echo "$revision" > "$STAMP_PATH"
+    if [[ -n "$FLUTTER_AOT_TOOL" ]]; then
+      echo Using dart2native...
```

Comment:
Should we echo to stderr (e.g. `>&2 echo 'Using dart2native...`) in case you're parsing the stdout. We parse `flutter devices --machine` in devicelab for example.

Reply:
Good idea. This isn't the only, echo - do you think it is safe to go through and make this change for the rest of them?

Reply:
Yes. I'm actually surprised, I thought I remember seeing a PR to move some echos to stderr because it was breaking someone's CI, but I don't see us redirecting to 2 anywhere in this file.

I can't think of any downside to redirecting all echos to STDERR other than that it could break tools/scripts grepping our stdout. And I don't think we should consider the stdout of our bash to be a stable API (plus it's an easy fix, read stderr)

Reply:
That having been said, I wouldn't be surprised if that change broke our own devicelab :P

Reply:
Only one way to find out :)